src/webservices.js

```diff
@@ -4,31 +4,37 @@ import config from "./config"
 
 
 function fetchJson(url, options) {
-  return fetch(url, options).then(parseJsonResponse)
+  return fetch(url, options)
+    .then(response => response.json()
+      .then(data => {
```

Comment:
The whole point of promises is that they can be chained. `return` the JSON and use `then` on the return value of `then`, not on `response.json()`.

[See more](http://stackoverflow.com/questions/22944717/callback-pyramid-even-with-when-promise).